index.js

```diff
@@ -146,7 +149,12 @@ function start (options, callback) {
 			selenium_proc.kill('SIGTERM');
 			selenium_proc.kill('SIGKILL');
 		}
-		callback && callback(err);
+		if (sauceConnectionRunning) {
+			console.log([moment().format('HH:mm:ss:SSS')] + ' e2e: something seems to have gone wrong, stopping sauce connect before travis shuts down');
+			stopSauceConnect(callback && callback(err));
```

Comment:
You need to pass in a function that itself calls the call back:
stopSauceConnect( function () {callback && callback(err)})